hacks/sh_status.js

```diff
@@ -22,6 +42,81 @@ printShardingStatus = function( configDB , verbose ){
         }
     );
 
+    // All of the balancer information functions below depend on a connection to a liveDB
+    // This isn't normally a problem, but can cause issues in testing and running with --nodb
+    if ( typeof db !== "undefined" ) {
+        output( "  balancer:" );
+
+        //Is the balancer currently enabled
+        output( "\tCurrently enabled:  " + ( sh.getBalancerState() ? colorize("yes", {color: "cyan"}) : colorize("no", {color: "red"}) ) );
```

Comment:
Nope. It became this long because of the `colorize()`:
https://github.com/mongodb/mongo/blob/v3.0/src/mongo/shell/shardingtest.js#L685

But if you see line 37 that's even longer.
Happy to split it in multiple lines if you want. What should be the max? 120 characters?